staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -42,7 +46,8 @@ import (
 
 var (
 	autoscaleLong = templates.LongDesc(i18n.T(`
-		Creates an autoscaler that automatically chooses and sets the number of pods that run in a Kubernetes cluster.
+		Creates an autoscaler that automatically chooses and sets the number of pods that run in a Kubernetes cluster using the autoscaling/v2 API version.
+        The command will attempt to use the autoscaling/v2 API first. If the autoscaling/v2 API is not available or an error occurs, it will fall back to the autoscaling/v1 API.
```

Comment:
```suggestion
        The command will attempt to use the autoscaling/v2 API first, in case of an error, it will fall back to  autoscaling/v1 API.
```